osu.Game/Overlays/Dashboard/Friends/FriendDisplay.cs

```diff
@@ -26,119 +26,104 @@ public List<User> Users
             set
             {
                 users = value;
-
                 onlineStreamControl.Populate(value);
```

Comment:
What happened to this thing? It doesn't seem to display anymore.

Reply:
probably the haphazard fillflow logic i removed. will need to be added back locally.